modular_skyrat/master_files/code/modules/awaymission/mission_code/black_mesa/mesa_mobs.dm

```diff
@@ -549,7 +549,7 @@
 	desc = "A 5.56mm surplus bullet casing."
 	projectile_type = /obj/projectile/bullet/a556/weak
 
-#define MOB_PLACER_RANGE CONFIG_GET(string/default_view) + 1 // One more tile than the biggest viewrange we have.
+#define MOB_PLACER_RANGE 16 // One more tile than the biggest viewrange we have.
```

Comment:
I dunno, I see issues down the line if TG ever added viewport adjustment or some of the downstreams edit their default config.